lib/graphql-anycable.rb

```diff
@@ -21,5 +22,13 @@ def redis
         ::AnyCable.broadcast_adapter.redis_conn
       end
     end
+
+    def config
+      @config ||= GraphQL::Anycable::Config.new
+    end
+
+    def configure
+      yield(config) if block_given?
+    end
```

Comment:
I can't find its usages. Do you use it in your application? How?

Reply:
I guess, that's just a common pattern for a global configuration. That's what people usually call in their initializers.

Reply:
That's what I have today:

```ruby
GraphQL::Subscriptions::AnyCableSubscriptions.class_eval do
  cattr_reader(:config) { GraphQL::Anycable::Config.new }

  config.subscription_expiration_seconds =
    Configby.graphql_api.subscription_expiration_seconds
end
```

_(I use `GraphQL::Subscriptions::AnyCableSubscriptions.config` in the copy-pasted cleaner Rake task code to share the config instance)_

That's what I can have with `configure`:

```ruby
GraphQL::Anycable.configure do |config|
  config.subscription_expiration_seconds = Configby.graphql_api.subscription_expiration_seconds
end
```